src/Workspaces/Remote/ServiceHub/Host/RemoteAnalyzerAssemblyLoader.cs

```diff
@@ -18,6 +18,7 @@ internal sealed class RemoteAnalyzerAssemblyLoader : AnalyzerAssemblyLoader
 
         public RemoteAnalyzerAssemblyLoader(string baseDirectory)
         {
+            // jason should we load from stream here on Linux or is this VS only?
```

Comment:
@jasonmalinowski I wanted you to look at this. The current state is always loading directly from disk (no shadow copy) and confirm it is the correct decision. If so I'll just leave this file alone (maybe add a comment to the effect). If this runs on Linux we could consider doing a stream based load here.

Reply:
This file isn't used on Linux, so your question may be somewhat moot. But the comment here makes me think that this kicks in if we're loading analyzers that shipped in Roslyn itself, so reading directly from disk is fine since there's not locking concerns in the first place.

Reply:
@genlu may also want to chime in here.

Reply:
Yes, the sole purpose of this is to load our own assemblies in ServiceHub process